---
layout: paper
title: Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2
image: /images/papers/kistler-sarscov2-adaptive-evolution.png
authors: Kistler KE, Huddleston J, Bedford T.
year: 2021
ref: Kistler et al. 2021. bioRxiv.
journal: "bioRxiv: 2021.09.11.459844."
pdf: /pdfs/papers/kistler-sarscov2-adaptive-evolution.pdf
doi: 10.1101/2021.09.11.459844
github: https://github.com/blab/sarscov2-adaptive-evolution
---

# Abstract

Despite the appearance of variant SARS-CoV-2 viruses with altered receptor-binding or antigenic phenotypes, traditional methods for detecting adaptive evolution from sequence data do not pick up strong signals of positive selection. Here, we present a new method for identifying adaptive evolution on short evolutionary time scales with densely-sampled populations. We apply this method to SARS-CoV-2 to perform a comprehensive analysis of adaptively-evolving regions of the genome. We find that spike S1 is a focal point of adaptive evolution, but also identify positively-selected mutations in other genes that are sculpting the evolutionary trajectory of SARS-CoV-2. Protein-coding mutations in S1 are temporally-clustered and, in 2021, the ratio of nonsynonymous to synonymous divergence in S1 is more than 4 times greater than in the equivalent influenza HA1 subunit.
